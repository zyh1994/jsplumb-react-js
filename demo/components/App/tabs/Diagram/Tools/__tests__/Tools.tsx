import React from 'react';
import renderer from 'react-test-renderer';
import Tools from '../Tools';

test('TabA tools', () => {
  const defaultFunc = () => { return; };

  const component = renderer.create(
    <Tools
      diagramId='id'
      onAddNode={defaultFunc}
      onRemoveNode={defaultFunc}
      onSetDiagramScale={defaultFunc}
      scale={0}
      selections={[]}
    />
  );
  const tools = component.toJSON();
  expect(tools).toMatchSnapshot();
});
