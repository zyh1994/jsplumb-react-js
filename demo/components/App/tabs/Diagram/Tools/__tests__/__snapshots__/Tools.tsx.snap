// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabA tools 1`] = `
<div
  className="jspr-tools"
>
  <h4>
    Complex Diagram Tools
  </h4>
  <button
    onClick={[Function]}
  >
    Add Node
  </button>
  <br />
  <br />
  <form>
    <label
      htmlFor="ui-orientation"
    >
      <input
        max="2"
        min="0.25"
        name="orientation"
        onChange={[Function]}
        step="any"
        type="range"
        value={0}
      />
      Scale 
      0
    </label>
  </form>
  <br />
  Selections:
  <code
    className="jspr-code"
  >
    [
    <br />
    ]
  </code>
  <br />
  <button
    onClick={[Function]}
  >
    Remove Nodes
  </button>
</div>
`;
